packages/pieces/apps/src/lib/clickup/actions/comments/get-task-comments.ts

```diff
@@ -0,0 +1,23 @@
+import { createAction, Property, HttpMethod, getAccessTokenOrThrow } from "@activepieces/framework";
+import { clickupCommon, callClickUpApi } from "../../common";
+
+
+export const getClickupTaskCommants = createAction({
+	name: 'get_task_comments',
+	description: 'Gets comments from a task in ClickUp',
+	displayName: 'Get Task Comments',
+	props: {
+		authentication: clickupCommon.authentication,
+		task_id: Property.ShortText({
+			description: 'The ID of the task to get',
+			displayName: 'Task ID',
+			required: true,
+		}),
+	},
+	async run(configValue) {
+		const { task_id, authentication } = configValue.propsValue;
+		return await callClickUpApi(HttpMethod.GET,
```

Comment:
I think it's better return body only, this applies for all others actions as well.

If there is an error, headers, body, status will be returned by activepieces http wrapper.

Reply:
Yep I agree with this, will update accordingly :)